src/int.cr

```diff
@@ -767,6 +767,17 @@ struct Int8
   MIN = -128_i8
   MAX =  127_i8
 
+  # Returns an `Int8` by invoking `to_i8` on *value*.
+  # See `String#to_i` for more details.
+  #
+  # ```
+  # Int8.new "20"                        # => 20
+  # Int8.new "  20  ", whitespace: false # => Unhandled exception: Invalid Int8:   20 (ArgumentError)
+  # ```
+  def self.new(value : String, base : Int = 10, whitespace : Bool = true, underscore : Bool = false, prefix : Bool = false, strict : Bool = true, leading_zero_is_octal : Bool = false) : self
+    value.to_i8 base, whitespace, underscore, prefix, strict, leading_zero_is_octal
```

Comment:
I am concerned that relying just on positions of these arguments has a risk of falling out of sync. Of course, that may not apply to Crystal, because changing their order would be an unacceptable breaking change anyway, but generally I'd always specify `leading_zero_is_octal: leading_zero_is_octal` etc.